airgun/views/hostgroup.py

```diff
@@ -117,7 +118,7 @@ class organizations(SatTab):
     @View.nested
     class activation_keys(SatTab):
         TAB_NAME = 'Activation Keys'
-        activation_keys = TextInput(id='kt_activation_keys')
+        activation_keys = TextInput(locator=".//div[contains(@class, 'rbt-input-multi')]")
```

Comment:
Dont we have an `id` for this multiselect ? Cause if we have another multiselect on this page it might conflict.

Element with `rbt-input-multi` class looks/might_be very common on the pages.

Reply:
@jyejare we do not have special id here but I did try come more closer locator, I hope it make sense for you.